frontend/src/components/worksheets/InformationModal/InformationModal.js

```diff
@@ -45,15 +45,15 @@ const KeyboardShortcutModal = ({ show, toggle }) => (
                     </tr>
                     <tr>
                         <td>
-                            <kbd>enter</kbd>
+                            <kbd>shift+enter</kbd>
```

Comment:
@michaeldu1 oops, I didn't realize that this PR was about switching keyboard shortcuts in the *new run* textbox, not the regular markdown textbox. In that case, we don't need to update the keyboard shortcuts for markdown.